homeassistant/components/media_player/apple_tv.py

```diff
@@ -258,7 +258,7 @@ def async_media_next_track(self):
         This method must be run in the event loop and returns a coroutine.
         """
         if self._playing:
-            return self.atv.remote_control.next()
+            return self.atv.remote_control.next()  # noqa: B305
```

Comment:
Can we disable B305 globally? It seems like a bit too eager.

Reply:
I had a look, but for some reason while adding `ignore = B306` to setup.cfg does actually disable that error as expected, after that a slew of /other/ errors that I think should have been reported before start to appear. I don't see them disabled anywhere (E123,E226 etc etc). I suspect a bug somewhere else unrelated to bugbear, will have a closer look unless someone beats me to it. (To reproduce, add an arbitrary ignore=foo line (or just ignore=) to setup.cfg's [flake8] section and run bare flake8 or tox -e lint).

Reply:
Ah, this seems to be documented behavior. Will have a look anyway.